partials/head.html

```diff
@@ -28,7 +28,3 @@
   href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined"
   rel="stylesheet"
 />
-<script>
-  const theme = localStorage.getItem("theme");
-  if (theme) document.documentElement.setAttribute("theme", theme);
-</script>
```

Comment:
If this is removed, the Theme-Loading will not work anymore. I will merge it anyways and refactor the Theme-Function in main.ts